frame/balances/src/lib.rs

```diff
@@ -560,6 +577,17 @@ pub struct BalanceLock<Balance> {
 	pub reasons: Reasons,
 }
 
+pub type ReserveIdentifier = [u8; 8];
```

Comment:
Or should I make this an associated type as well and define it in runtime?

Reply:
can do; will save a little bit of work later.